recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ build:
   skip: True  # [py<38]
   # s390x is missing pyrarrow that datasets depends on
   skip: True  # [linux and s390x]
-  number: 0
+  number: 1
```

Comment:
Should the build number be changed since no other changes have been made?

Reply:
It's debatable. Given the type of dependencies, no compiler, no libraries in host, keeping the build number and relying on skip-existing would be sufficient in my opinion.